src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1746,6 +1746,19 @@ bool ApplicationManagerImpl::ManageMobileCommand(
   }
 
   if (message_type == mobile_apis::messageType::response) {
+    if (!message->isValid()) {
+      LOG4CXX_ERROR(logger_, "RET Mobile message response is invalid");
```

Comment:
@AKalinich-Luxoft What is `RET` ?

Reply:
@AByzhynar I suppose it means RETURN from function with some exceptional case and a reason of return. This RET is placed in all exceptional cases inside ApplicationManagerImpl::ManageMobileCommand

Reply:
@AByzhynar removed in https://github.com/smartdevicelink/sdl_core/pull/1787/commits/59af9ea67c0acdeb85ffe39e65eafbd321bd01a3